samples/model-builder/test_constants.py

```diff
@@ -103,6 +104,7 @@
 MACHINE_TYPE = "n1-standard-4"
 ACCELERATOR_TYPE = "ACCELERATOR_TYPE_UNSPECIFIED"
 ACCELERATOR_COUNT = 0
+ONLINE_STORE_FIXED_NODE_COUNT = 1
```

Comment:
Create a section for feature store constants

Reply:
Sounds good